docs/cmdlet-example/visual-studio-simple-example.md

```diff
@@ -0,0 +1,125 @@
+            {
+                WriteObject("Hello " + Name + "!");
+            }
+        }
+    }
+    ```
+    At this point everything should look like this:
+    ![Step5](./Images/Step5.png)
+
+1. Build solution (F6); The `Output` window will print the location of generated cmdlet DLL:
+  ![Step6](./Images/Step6.png)
+
+1. Start PowerShell Core, run `Import-Module` on DLL path from previous step and run cmdlet:
+  ![Step7](./Images/Step7.png)
+You can also run the same cmdlet on Linux and other systems that PowerShell Core supports:
+![Step8](./Images/Step8.png)
+
+## Building a C# Cmdlet for PowerShell Standard 3.0 with Visual Studio
+
+Steps below show how to build your own C# cmdlet for PowerShell Standard 3.0 with Visual Studio.
+
+We will use the free [Visual Studio Community 2017](https://www.visualstudio.com/downloads).
+
+1. When installing Visual Studio 2017 select `.NET Core cross-platform development` under `Other Toolsets`
+  ![StdImage1](./Images/Step1.png)
+
+1. Create new C# project `SendGreetingStd` of type `Class Library (.NET Standard)`
+  ![StdImage21](./Images/Std21.png)
+On project properties verify that `Target framework` is `.NET Standard 2.0`:
+![StdImage22](./Images/Std22.png)
+
+1. Now we need to setup reference assemblies.
+  In `Solution Explorer` right click on project `Dependencies` and select `Manage NuGet Packages...`
+In the top-right corner of the package manager select `nuget.org` package source, select `Browse` tab, type in `PowerShellStandard.Library` in the search and select `Include prerelease`.
+It should find `PowerShellStandard.Library` package, select it and it will show package details; install it using `Install` button.
+![StdImage3](./Images/Std3.png)
+
+1. Add the code of cmdlet:
+    ```CSharp
+    using System.Management.Automation;  // PowerShell namespace.
+
+    namespace SendGreeting
+    {
+        // Declare the class as a cmdlet and specify and
+        // appropriate verb and noun for the cmdlet name.
+        [Cmdlet(VerbsCommunications.Send, "Greeting")]
+        public class SendGreetingCommand : Cmdlet
+        {
+            // Declare the parameters for the cmdlet.
+            [Parameter(Mandatory = true)]
+            public string Name { get; set; }
+
+            // Overide the ProcessRecord method to process
+            // the supplied user name and write out a
+            // greeting to the user by calling the WriteObject
+            // method.
+            protected override void ProcessRecord()
+            {
+                WriteObject("Hello " + Name + "!");
+            }
+        }
+    }
+    ```
+    At this point everything should look like this:
+    ![StdImage4](./Images/Std4.png)
+
+1. Build solution (F6); The `Output` window will print the location of generated cmdlet DLL:
+  ![StdImage5](./Images/Std5.png)
+
+1. Now cmdlet can be run on systems supported by PowerShell Standard; for example:
+  On PowerShell Core on Windows:
```

Comment:
Is this line supposed to be on a newline? It's rendered in the same line as above text.

Reply:
Fixed.